README.md

```diff
@@ -2,38 +2,45 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-userservices-dns/workflows/build/badge.svg)](https://github.com/cisagov/cool-userservices-dns/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
+This is a Terraform deployment to create DNS-related resources needed for
+services in the COOL User Services account.
+
+## Pre-requisites ##
+
+- [Terraform](https://www.terraform.io/) installed on your system.
+- An accessible AWS S3 bucket to store Terraform state
+  (specified in [backend.tf](backend.tf)).
+- An accessible AWS DynamoDB database to store the Terraform state lock
+  (specified in [backend.tf](backend.tf)).
+- Access to all of the Terraform remote states specified in
+  [remote_states.tf](remote_states.tf).
+- The following COOL accounts and roles must have been created:
+  - Master:
+    [`cisagov/cool-accounts/master`](https://github.com/cisagov/cool-accounts/master)
+  - Terraform:
+    [`cisagov/cool-accounts/terraform`](https://github.com/cisagov/cool-accounts/terraform)
+  - User Services:
+    [`cisagov/cool-accounts-userservices`](https://github.com/cisagov/cool-accounts-userservices)
+  - Users:
+    [`cisagov/cool-accounts/users`](https://github.com/cisagov/cool-accounts/users)
+- Terraform in [`cisagov/cool-dns-cyber.dhs.gov`](https://github.com/cisagov/cool-dns-cyber.dhs.gov)
+  must have been applied.
+- Terraform in [`cisagov/cool-userservices-networking`](https://github.com/cisagov/cool-userservices-networking)
+  must have been applied.
```

Comment:
I'm not sure that these last three bullets are still necessary, but I assume you are leaving them in for when there is some real content for this repo.  I think a sentence explaining that will help remind us to update accordingly when that time comes.

Reply:
You're correct, though I think it's cleaner to just excise them now and add them back later when they are actually required.  See https://github.com/cisagov/cool-userservices-dns/pull/2/commits/0ad2879374c354f14017ac23559bbdc4fad3abd6 and https://github.com/cisagov/cool-userservices-dns/pull/2/commits/2152fffc3295e95681fea72d7bc8920146a5e161.

The "Users" account is a pre-requisite here because `cisagov/terraform-state-read-role-tf-module` creates an assume role in the "Users" account: https://github.com/cisagov/cool-userservices-dns/blob/0ad2879374c354f14017ac23559bbdc4fad3abd6/read_terraform_state_role.tf#L7-L20